src/handlers/vlc.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file is part of usysconf.
+ *
+ * Copyright © 2017-2019 Solus Project
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+
+#define _GNU_SOURCE
+
+#include "context.h"
+#include "files.h"
+#include "util.h"
+
+static const char *vlc_modules_paths[] = {
+        "/usr/lib/vlc/plugins/",
+        "/usr/lib64/vlc/plugins/",
+};
+
+/**
+ * Create a VLC plugin cache
+ */
+static UscHandlerStatus usc_handler_vlc_exec(__usc_unused__ UscContext *ctx, const char *path)
```

Comment:
`ctx` does appear to actually be used.
```suggestion
static UscHandlerStatus usc_handler_vlc_exec(UscContext *ctx, const char *path)
```